### DISCUSSION TITLE
REOLINK TRACKMIX POE PTZ  does not seem to work

### DISCUSSION THREAD

**Discussion by eon0613** [blakeblackshear/frigate#10641] [General]
I added the ONVIF parameters on my CONFIG.YML but when i open the FRIGATE WEB UI, the CONTROL PANEL  is just spinning.
Did i miss something on the config or REOLINK TRACKMIX is not supported.

CONFIG.YMLcameras:
  front_yard:
    ffmpeg:
      inputs:
      - path: rtsp://127.0.0.1:8554/front_yard
        input_args: preset-rtsp-restream
        roles:
        - detect  
    onvif:
      host: 192.168.1.99
      port: 8000
      user: admin
      password: Leianne@1994    

thanks in advance

**Comment by NickM-27** [comment]
I believe I saw a user say that the trackmix doing manual onvif interfered with its default tracking behavior and caused issues

**Reply by eon0613** [reply]
ok thanks again for your quick response
